binaries/scheduler/config/local.local

```diff
@@ -12,7 +12,9 @@
     "Type": "stateful",
     "MaxRetriesPerTask" : 2,
     "DebugMode" : false,
-    "RecoverJobsOnStartup" : true
+    "RecoverJobsOnStartup" : true,
+    "DefaultTaskTimeoutMs" : 1800000,
+    "RunnerOverheadMs": 600000
```

Comment:
10 minutes of overhead accounts for you hitting a worker that's just starting up and needs to initialize.